### PR TITLE
Automatic commit: Move 'cloudsdk-vdm-generator' from QA to Production

### PR DESCRIPTION
**Pull request by karstenmelato**
Update Authors. Please pull to production.

### CODE REVIEW

File: tutorials/cloudsdk-vdm-generator/cloudsdk-vdm-generator.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+            get("/accounts").
+            then().
+            statusCode(200).
+            contentType(ContentType.JSON).
+            body("$", hasSize(greaterThan(0))).
+            body("[0].Provider", not(isEmptyOrNullString()));
+}
+```
+
+Please, note that you will need `systems.json` and `credentials.yml` files in the resources folder in your integration tests to mock `ErpDestination` in tests. Please, refer to the blog [Step 13 with SAP Cloud SDK: Automated Testing](https://blogs.sap.com/2017/09/19/step-12-with-sap-s4hana-cloud-sdk-automated-testing/) section Mock ERP Destination for more details on this topic.
```
Comment (thecodester) [inline tutorials/cloudsdk-vdm-generator/cloudsdk-vdm-generator.md:346]: No need for comma here -- please take another quick edit through the tutorial.

File: tutorials/cloudsdk-vdm-generator/cloudsdk-vdm-generator.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+
+    The blog [Deep Dive 1 with SAP Cloud SDK: An End-to-End Story on the Virtual Data Model using the Business Partner API](https://blogs.sap.com/2017/09/15/step-12-with-sap-s4hana-cloud-sdk-consuming-business-api-from-s4hana-via-odata/) provides a good practical introduction on how to create the communication system, user, and arrangement in order to enable access to APIs from the local machine, which is sufficient for the example given in this blog post.
+
+    Also, please refer to this [blog post](https://blogs.sap.com/2017/09/15/what-is-what-your-quick-reference-to-communication-management-and-identity-access-management-artifacts-in-s4hana/) to get overview on the concept of SAP S/4HANA Cloud communication and integration artifacts.
+
+5. Evaluate the generated OData service
+
+    Now, when we have enabled the generated custom OData service, let us take a look at its metadata. The metadata of the OData service is represented by an `edmx` file that can be accessed by adding /$metadata to the OData service base path. We use the following URL:
```
Comment (thecodester) [inline tutorials/cloudsdk-vdm-generator/cloudsdk-vdm-generator.md:150]: /$metadata should be in back ticks -- as well as code, files, folders, URLS. Please take a look through the tutorial for other cases.

**Comment by thecodester** [comment]
@karstenmelato Are you still interested in this PR?

**Comment by MichaelCzcz** [comment]
@karstenmelato Since you didn't bother to respond to neither Daniels's question from 19th May nor to my questions sent via email sent on 22nd July 2020 and again on 16th September, if you are still interested in this PR getting merged, I am closing this PR.

**Pull request closed without merging**